dxcherry/src/github/createPullRequest.ts

```diff
@@ -1,7 +1,8 @@
 import GithubClient from './client';
 import { logInfo } from '../info';
+import { TreePayload } from '../tree/payload';
 
-async function createPullRequest (login: string, branchWithoutVersion: string, version: string, title: string, description: string, labels: string[], reviewers: string[]) {
+async function createPullRequest ({ title, description, labels, reviewers}: TreePayload, login: string, branchWithoutVersion: string, version: string) {
```

Comment:
Remove destructuring from processCherryPickRequest and pass the whole TreePayload object to this function. 
Create branchName (`${login}:${branch}_${version}`) here and remove this logic from GithubClient.createPullRequest. Pass there only branchName, title and description.

Reply:
there is already no destructuring in `processCherryPickRequest` now.

We cannot pass only branchName to `GithubClient..createPullRequest` because of `version` using. But we can can remove one excess parameter - I made it